samples/advanced/data-labelling.html

```diff
@@ -3,130 +3,130 @@
 <html>
 
 <head>
-    <title>Labelling Data Points</title>
-    <script src="../../dist/Chart.bundle.js"></script>
-    <script src="../utils.js"></script>
-    <style>
-    canvas {
-        -moz-user-select: none;
-        -webkit-user-select: none;
-        -ms-user-select: none;
-    }
-    </style>
+		<title>Labelling Data Points</title>
```

Comment:
All files are now indented with 2 tabs instead of 1 for the HTML part

![image](https://user-images.githubusercontent.com/3874900/35537283-f7136e6c-0549-11e8-897d-3111c8d391f6.png)